src/main/java/core/basesyntax/model/FigureSupplier.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax.model;
+
+import core.basesyntax.enums.AvailableFigure;
+import core.basesyntax.enums.Color;
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final String DEFAULT_COLOR = Color.WHITE.getColor();
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int MAX_SIDE_UNITS_NUMBER = 100;
+    private static final int MIN_SIDE_UNITS_NUMBER = 1;
+    private final AvailableFigure[] availableFigures = AvailableFigure.values();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        switch (availableFigures[random.nextInt(availableFigures.length)]) {
```

Comment:
Generally, it's not considered good practice to place such a long expression inside the switch. It's better to extract it to a variable and name it somehow to give more context on what this expression actually does.